### DISCUSSION TITLE
Integrating TCP/IP using GPRS Module

### DISCUSSION THREAD

**Discussion by 00bear** [miguelgrinberg/python-socketio#867] [Q&A]
Great Project! However using Flask on my project seemed to be impossible at first because particular client needed too many resources; it's not really the case anymore. Could we connect directly to socket over 2G network and push/pull data to database using AT commands ? I realize I'm still not asking to keep any data server side.

**Comment by miguelgrinberg** [comment]
Please do not write issues on this project when you have a question. The discussion board is for that. The issues board is for bugs and enhancements only.

This project does not depend on Flask. Also, are you sure you are in the right project? Socket.IO is a high-level protocol, it is not a bare network socket. I think you must be looking for something else.

**Reply by 00bear** [reply]
Thanks! FOUND IT. delete
